packages/multichain/src/caip25Permission.test.ts

```diff
@@ -265,8 +308,40 @@ describe('caip25EndowmentBuilder', () => {
         });
       });
 
+      it('revokes the permission if the only account is removed', () => {
```

Comment:
should we have one:
"does not revoke the permission if the only remaining accounts are in the `wallet:eip155` scope"?

Reply:
sure why not https://github.com/MetaMask/core/pull/4978/commits/402e3f8a38829e7c3e351c3cbefc9dc77949a93c